apps/react/src/features/utils/date-format.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Date format. Ex: October 7, 2020.
+ * @param date Date. Ex: Wed Oct 07 2020 07:00:00 GMT+0700.
+ */
```

Comment:
Let's improve docs a bit 
```suggestion
/**
 * Formats JS date to a string with month, day and year.
 * @param date Date (e.g. Wed Oct 07 2020 07:00:00 GMT+0700).
 * @example
 * ```
 * dateFormat(startDate) => October 7, 2020
 * ```
 */
```